php/classes/class-qmn-quiz-manager.php

```diff
@@ -1266,6 +1266,13 @@ public function check_answers($options, $quiz_data) {
                         if ($page_question_id == $question_id) {
 
                             $question = $questions[$page_question_id];
+                            // Ignore non points questions from result
+                            $question_type_new = $question['question_type_new'];
+                            if($question_type_new != 0 && $question_type_new != 1 && $question_type_new != 2 && $question_type_new != 4
```

Comment:
can you refactor this for more clarity? Also I am not sure why you are using AND logic. Maybe I am missing something but feels like there should be OR in logic. And for multiple checks can you can use something like `in_array() `